src/components/editions/kickerPicker.ts

```diff
@@ -0,0 +1,60 @@
+import type { Tag } from '@guardian/content-api-models/v1/tag';
+import { TagType } from '@guardian/content-api-models/v1/tagType';
+import type { Option } from '@guardian/types';
+import { none, some } from '@guardian/types';
+import type { Item } from 'item';
+
+const tagTitleIsAlreadyInHeadline = (tag: Tag, headline: string): boolean =>
+	headline.toLowerCase().includes(tag.webTitle.toLowerCase());
+
+/*
+ - Use series tag if we have one
+ - Use tone for {"Letters", "Analysis", "Obituaries"}
+ - Use tone without the last character for {"Reviews", "Editorials", "Match Reports", "Explainers"}
+ - Use byline for "Comment" tone
+ - Use top tag if both top and second tag feature in the headline
+ - Use second tag if top tag features in the headline
+ - Otherwise use top tag
+ */
+const kickerPicker = (article: Item): Option<string> => {
+	const byline = article.byline;
+	const seriesTag = article.tags.find((tag) => tag.type === TagType.SERIES);
+	const toneTag = article.tags.find((tag) => tag.type === TagType.TONE);
+
+	if (seriesTag) return some(seriesTag.webTitle);
+
+	if (toneTag?.id) {
+		if (
+			toneTag.id === 'tone/letters' ||
+			toneTag.id === 'tone/analysis' ||
+			toneTag.id === 'tone/obituaries'
+		)
+			return some(toneTag.webTitle);
+
+		if (
+			toneTag.id === 'tone/reviews' ||
+			toneTag.id === 'tone/editorials' ||
+			toneTag.id === 'tone/matchreports' ||
+			toneTag.id === 'tone/explainers'
+		)
+			return some(toneTag.webTitle.slice(0, -1));
+
+		if (toneTag.id === 'tone/comment') return some(byline);
+	}
+
+	const topTag = article.tags[0] as Tag | undefined;
+	const secondTag = article.tags[1] as Tag | undefined;
```

Comment:
@JamieB-gu Then I get a TS error `Unnecessary conditional, value is always falsy` on line 48 and 52. Which I don't understand why ...  🤔 If you have any insights I am most intrigued!

Reply:
That's odd... I don't get that. However I see that it's sensible to mark these as `undefined`. Indexed lookups on arrays can technically return `undefined`, but TS won't throw a compiler error 😔. Can I recommend a type annotation instead of a type cast?

```suggestion
	const topTag: Tag | undefined = article.tags[0];
	const secondTag: Tag | undefined = article.tags[1];
```

Annotations work with the type-checker, whereas casts override it.

Reply:
@JamieB-gu Same error happens, I think that's why I eventually settled on this type cast which wouldn't be my first choice either .. 
Realising though it's not TS that's unhappy but the linter 😑

Reply:
Interesting, it seems that, even with the annotation, TS continues to type them both as `Tag` further down the file. So I think the linter's right here, you have to type cast them sadly. Consequence of a design decision by TS not to type array lookups as possibly `undefined`.

Reply:
Here's an alternative which also comes with the benefit of converting to an `Option`, which you're returning from the function later anyway:

```suggestion
	const topTag = fromNullable(article.tags[0]);
	const secondTag = fromNullable(article.tags[1]);
```

`topTab` and `secondTag` will both become `Option<Tag>`.